lib/wanda_web/views/health_view.ex

```diff
@@ -0,0 +1,10 @@
+defmodule WandaWeb.HealthView do
+  use WandaWeb, :view
```

Comment:
Now hardly will break, but can we have a test on this?

Reply:
Added